### PR TITLE
Banner and readme update

### PR DESCRIPTION
**Pull request by shaloo**
# PR

## Describe your changes

Banner update, readme cleanup

## Issue ticket number and link

- [AR-7633](https://team-1624093970686.atlassian.net/browse/AR-7633)

## Checklist before requesting a review

- [x] You have performed a self-review of your own code
- [x] You are using approved terminology
- [x] Your changes have been tested locally
- [x] Your code builds clean without any errors or warnings

[AR-7633]: https://team-1624093970686.atlassian.net/browse/AR-7633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by anivenu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,18 +11,16 @@
...
 </p><p id="start" align="center">
-<a href="https://docs.beta.arcana.network/"><img src="https://raw.githubusercontent.com/arcana-network/branding/main/an_banner_docs.png" alt="Arcana Auth SDK"/></a>
+<a href="https://docs.beta.arcana.network/"><img src="https://raw.githubusercontent.com/arcana-network/branding/main/an_auth_sdk_banner_feb_24.png" alt="Arcana Auth SDK"/></a>
 </p>
 
-# Arcana Auth
+# Auth
 
-Use Arcana Auth SDK to onboard Web3 app users via social login and allow authenticated users to instantly access the built-in, embedded Arcana wallet and sign blockchain transactions from within the app context. 
+[Arcana Network](https://arcana.network) Auth SDK integrates with Web3 apps to enable social login and embedded, in-app Arcana wallet. Users don't need to install a wallet. Developers can leverage the built-in gasless feature to sponsor gas fees for white-listed blockchain transactions.
```
Comment (mmjee) [inline README.md]: Isn't it the other way around? Web3 applications can integrate with the Arcana Network Auth SDK to enable social logins, etc.?

File: README.md [hunk 2]
```diff
@@ -40,25 +38,37 @@ Auth SDK enables the standard [Ethereum provider](https://eips.ethereum.org/EIPS
 
 - Passwordless authentication
 
-Auth SDK usage is easily customizable. Developers can choose to use the built-in plug and play login UI to onboard users or use a custom UI. Also, instead of the built-in Arcana wallet UI, developers can integrate with the SDK and use a custom wallet UI.
+## Supported Blockchain Networks
+
+The Arcana wallet is [pre-configured with a subset of supported chains](https://docs.arcana.network/state-of-the-arcana-auth#supported-blockchains). Others can be added programmatically or via the wallet UI.
```
Comment (mmjee) [inline README.md]: The dashboard?

File: README.md (hunk 1 above)
Comment (shaloo) [inline README.md]:
```suggestion
Web3 apps can integrate with the [Arcana Network](https://arcana.network) Auth SDK to enable social login and embedded, in-app Arcana wallet. Users don't need to install a wallet. Developers can leverage the built-in gasless feature to sponsor gas fees for white-listed blockchain transactions.

File: README.md (hunk 2 above)
Comment (shaloo) [reply README.md]: hmm, yes but by the devs. Ok let me rephrase it.

Comment (shaloo) [inline README.md]:
```suggestion
The Arcana wallet is [pre-configured with a subset of supported chains](https://docs.arcana.network/state-of-the-arcana-auth#supported-blockchains). Other supported chains can be added by app developers programmatically or via the dashboard. App users can add supported chains through the wallet UI.
